styleguide/_themes/derek/scss/components/_rug.scss

```diff
@@ -1,15 +1,18 @@
-.rug-phone-numbers{
-  margin-top:6px;
-}
-.rug-subhead{
-  font-size: 37px;
-  line-height: 24px;
+.rug-contact-link {
+  padding-left: 5px;
+  text-decoration: none;
```

Comment:
Alternatively, DP recommended doing:

```
.rug-contact-link-hover {
  &:hover, &:focus {
    color: $white;
    text-decoration: underline;
  }
}
```

and then adding the class to the HTML structure.